en/telemetry/esp32_wifi_module.md

```diff
@@ -1,169 +1,159 @@
-   Make sure the voltage levels match: most ESP32 DevKits can only take 3.3V!
-   - TX to RX
-   - RX to TX
-   - GND to GND
-   - Provide a stable 3.3V or 5V power supply to the ESP32 (depending on the available inputs of your DevKit)
-1. Set the flight controller port to the desired protocol.
-1. Boards with an IPEX port for an external antenna often also offer an onboard antenna that is activated by default.
-   You may need to resolder a resistor to activate the external antenna port.
+- Connect UART of ESP32 to a UART of your autopilot (ie. `TELEM1` or `TELEM2` port).
+  Make sure the voltage levels match!
+- RX to TX
+- GND to GND
+- Stable 3.3V or 5V power supply to the ESP32 (depending on the available inputs of your ESP32 and capabilities of the autopilot)
+- Set the autopilot port to MAVLINK 1 or 2 protocol.
 
 ::: info
 
-- Follow the ESP32 board manufacturers recommendations on power supply.
+- Follow the ESP32 board manufacturer recommendations on power supply.
   Some boards might have issues if they are simultaneously connected to a 5V power source and have a USB cable connected to the USB/Serial bridge (USB socket of the ESP32 dev board).
 - Some ESP32 DevKits manufacturers use the wrong labels for the pins on their products.
-  Make sure that the PINs on your board are labeled correctly if you encounter issues.
+  Make sure that the PINs on your board are labelled correctly if you encounter issues.
 
 :::
 
-## Configuring QGroundControl
+## Configuring PX4
 
-QGroundControl should auto-detect the connection and no further actions should be necessary (_DroneBridge for ESP32_ automatically forwards data from all connected WiFi devices via UDP to port 14550).
+Configure the Flight controller port that is wired to the ESP32 to have MAVLink set as the protocol and to have the same baud rate as the ESP32.
+The Pixhawk `TELEM1` and `TELEM2` ports are configured for [MAVLink by default](../peripherals/mavlink_peripherals.md#default-mavlink-ports) but you will still need to set the rates.
 
-The following connection options are available:
+For `TELEM1` you might set the values like this:
 
-- UDP unicast on port `14550` to all connected devices.
-- TCP on port `5760`
+- `SER_TEL1_BAUD`: 115200 8N1
+- `MAV_0_RATE`: 24000 (can be increased optionally)
+
+For more information, and using other ports, see [MAVLink Peripherals](../peripherals/mavlink_peripherals.md) and [Serial Port Configuration](../peripherals/serial_configuration.md)
 
 ## Configuring DroneBridge for ESP32
 
 The _DroneBridge for ESP32_ default configuration should work for connecting to PX4 "out of the box".
+For the Official HW1.x Boards the configuration is: `UART TX Pin: 5`, `UART RX Pin: 4`, `UART RTS Pin: 6`, `UART CTS Pin: 7`
+
 The only configuration that may be required is ensuring that the baud rates of the ESP32 and flight controller match.
 
+![DroneBridge for ESP32 Webinterface](../../assets/peripherals/telemetry/esp32/dbesp32_webinterface.png)
```

Comment:
Again, how do I get this interface? Direct link please.

Reply:
Added description. Web interface can be accessed when connected to the access point that is launched once the ESP32 is powered on.